Formula/figtree.rb

```diff
@@ -0,0 +1,21 @@
+class Figtree < Formula
+  desc "Graphical viewer of phylogenetic trees"
+  homepage "http://tree.bio.ed.ac.uk/software/figtree/"
+  url "https://github.com/rambaut/figtree/releases/download/v1.4.3/FigTree_v1.4.3.tgz"
+  sha256 "f497d4dd3a6d220f6b62495b6f47a12ade50d87dbd8d6089f168e94d202f937b"
+
+  # tag "bioinformatics"
+
+  depends_on :java
+
+  def install
+    prefix.install "lib/figtree.jar", "images"
+    bin.write_jar_script prefix/"figtree.jar", "figtree"
+    pkgshare.install Dir["*.tree"]
+    doc.install "README.txt"
```

Comment:
This file should be installed automagically. Files whose base name without the extension is one of the following are installed automatically.
```
    about authors changelog changes copying copyright history license licence
    news notes notice readme todo
```
See https://github.com/Linuxbrew/brew/blob/master/Library/Homebrew/metafiles.rb#L10